Lib/test/test_grammar.py

```diff
@@ -383,9 +383,10 @@ def test_var_annot_simple_exec(self):
         gns = {}; lns = {}
         exec("'docstring'\n"
              "x: int = 5\n", gns, lns)
+        self.assertNotIn('__annotate__', gns)
+
+        gns.update(lns)  # __annotate__ looks at globals
         self.assertEqual(lns["__annotate__"](annotationlib.Format.VALUE), {'x': int})
```

Comment:
This test started failing because `__annotate__` now always accesses `__conditional_annotations__` from the globals, and the separate globals and locals namespaces break that. This would have already failed if the annotations referred to any name defined in the module (e.g. `type x = int; y: x`), so I don't think that's a problem.